core/src/main/resources/error/error-classes.json

```
@@ -1209,6 +1226,23 @@
     ],
     "sqlState" : "22032"
   },
+  "INVALID_LAMBDA_FUNCTION_CALL" : {
+    "message" : [
+      "Invalid lambda function call."
+    ],
+    "subClass" : {
+      "ARGUMENT_NAMES_EXISTS_SAME_SEMANTIC" : {
+        "message" : [
+          "Lambda function arguments should not have names that are semantically the same."
```

From my point of view, the message is overcomplicated. What does `semantically` mean?
Let's make it more precise:
```suggestion
          "The lambda function has duplicate arguments <args>. Please, consider to rename the argument names or set <caseSensitiveConfig> to \"true\"."
```